test/unit/dash.DashAdapter.js

```diff
@@ -59,11 +59,24 @@ const manifest_with_ll_service_description = {
         }]
     }]
 };
-const manifest_without_supplemental_properties = {
+const manifest_without_properties = {
     loadedTime: new Date(),
     mediaPresentationDuration: 10,
     Period: [{ AdaptationSet: [{ id: 0, mimeType: Constants.VIDEO }] }]
 };
+const manifest_with_essential_properties = {
+    loadedTime: new Date(),
+    mediaPresentationDuration: 10,
+    Period: [{
+        AdaptationSet: [{
+            id: 0, mimeType: Constants.VIDEO,
+            EssentialProperty: [{
```

Comment:
Also use `DashConstants.ESSENTIAL_PROPERTY` here?

Reply:
I see your point.

However, why not using `DashConstants.PERIOD`, `DashConstants.ADAPTATION_SET`,... in the test files?

IMHO this would be a major change to all (unit?) tests. We could do so, but that's a task of its own.